### ISSUE TITLE
Check that encoders work

### ISSUE THREAD

**Issue by jzkyu** [Patriotic-Robotics-6372/SkyStone#1]
Using the encoder method included in the FTC Programming Guide does not work as intended; we will instead look into the javadoc and resources online to better understand how encoders work

**Comment by jzkyu** [comment]
What we tried: We copied the code from the programming guide, reframing it to our motor names. It seems to work sometimes. 
- The behavior: sometimes it goes forward and goes to a complete stop as intended. But other times it will keep on running, as the left side seems to count slower on the encoders. Because of this, the while loop conditional keeps running for much longer than it should. 
- What we can do: Change the motors that use encoders from frontLeft to backLeft, and see if the encoder count goes up faster as intended. OR: Only use the frontRight motor encoders and see if the issue persist. 
- Implications: The frontLeft motor possibly needs to be replaced. But we'd need to be absolutely sure of this before doing so
